src/Helpers/BodyLimit.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Dumbo\Helpers;
+
+use Dumbo\Context;
+
+class BodyLimit
+{
+    public static function limit(int $maxSize, callable $onError = null): callable
```

Comment:
Missing the doc block. Something like this?

```php
/**
     * Create a middleware that limits the size of the request body
     *
     * @param int $maxSize The maximum allowed size of the request body in bytes
     * @param callable|null $onError Custom error handler (optional)
     * @return callable The middleware
     */
```

Reply:
Ah yes!